test/deploy/k8s/cluster.yaml

```diff
@@ -44,15 +44,10 @@ cluster:
         dix: 1+7+0
       nodes:
         - name: podnode-0
-          id: 1111
-          hostname: pod-1.colo.seagate.com
+          id: 285d6133-3645-46b7-96a7-56917cd3cb43
+          hostname: storagenode.colo.seagate.com
```

Comment:
Not sure if it's right in this context, but see if  it's possible to avoid references to the seagate domain name/server names.

Reply:
For now, these files are used only for Our testing and will not be used outside Seagate network. The Cortx-services team will take care of generating these files.